dlt-database/migrations/0003-refactor_transaction_recipe.ts

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+export async function upgrade(queryFn: (query: string, values?: any[]) => Promise<Array<any>>) {
+  // write upgrade logic as parameter of queryFn
+  await queryFn(`DROP TABLE \`confirmed_transactions\`;`)
+  await queryFn(`DROP TABLE \`transaction_recipes\`;`)
+
+  await queryFn(`
+    ALTER TABLE \`accounts\` 
+      RENAME COLUMN \`balance\` TO \`balance_confirmed_at\`,
```

Comment:
```suggestion
      RENAME COLUMN \`balance\` TO \`balance_on_confirmation\`,
```

Reply:
following our existing naming conventions the postfix "_at" always suggerats a datetime field.
Here you want to define the field for the balance, which contains the balance-amount at the point of time on confirmation, right?

Reply:
correct, I like you suggestion, I update my code.